server/controllers/getProductsByCId.js

```diff
@@ -0,0 +1,13 @@
+const {productByCategory} = require('../database/queries');
+
+
+
+const getProductByCId = (req, res) => {
+    const category_id = req.params;
+
+    productByCategory(cid)
```

Comment:
I think you changed this in the top
```suggestion
    productByCategory(categoryId)
```